setup.sh

```diff
@@ -36,7 +36,7 @@ while [[ $# -gt 0 ]]; do
 done
 
 if [[ -z "$repos" ]]; then
-  repos="worker server jobs pgbouncer data-model"
+  repos="worker server jobs pgbouncer data-model stack-analysis firehose-fetcher scaler recommender"
```

Comment:
We have also [worker-base](https://github.com/fabric8-analytics/fabric8-analytics-worker-base)